<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCentralWidget</class>
 <widget class="QWidget" name="GameCentralWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>495</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Chalkboard SE</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:black}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkduster</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background:yellow;border:none}</string>
   </property>
   <property name="text">
    <string>START!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phosphate</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:yellow}</string>
   </property>
   <property name="text">
    <string>First Hand</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkduster</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:yellow}</string>
   </property>
   <property name="text">
    <string>Your Scroe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>350</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:black}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:black}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color:black}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkduster</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: yellow;border:none}</string>
   </property>
   <property name="text">
    <string>Back to Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkduster</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color: rgba(0,0,0,0%)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkduster</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color: rgba(0,0,0,0%)}</string>
   </property>
   <property name="text">
    <string>Type number you want to ask:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>164</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDialogButtonBox{border:yes;border-color:yellow}</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkduster</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: rgba(0,0,0,0%);border:yes;}</string>
   </property>
   <property name="text">
    <string>Ready?Click to show your card!</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDialogButtonBox{border:yes;border-color:yellow}</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>back_to_menu()</slot>
 </slots>
</ui>
